api/types/updatable_project_fields.go

```diff
@@ -47,6 +48,27 @@ var (
 	nameRegex = regexp.MustCompile("^[a-z0-9\\-._~]+$")
 )
 
+// ErrorWithDetails is error for deliver details with error
+type ErrorWithDetails struct {
+	err     error
+	details interface{}
```

Comment:
Can we use generics here?

Reply:
Is there a dependency issue on protobuf in the picture above?